### ISSUE TITLE
Vomiting from coughing spell

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9780]
**Preferred term label:**
Vomiting from coughing spell

**Synonyms**
Vomiting from elevation in intrabdominal pressure

**Definition (free text, please give PubMed ID)**
Individuals with GERD will frequently vomit when coughing for a non-GI reason.  Vomiting occurs during these coughing spells because the intrabdominal pressure is high from the coughing and the propensity to reflux results in stomach contents emerging as vomit.  This is a historical question that allows a parent to impart information suggesting GERD at an age when it is difficult to obtain a reliable GERD history from the child.  

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
GERD

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
Better name is posttussive emesis

**Comment by pnrobinson** [comment]
@MickeySegal 

Added as
Posttussive emesis
HP:0034783
